system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -262,8 +262,11 @@ class XpathContextXmlWriter : public CInterfaceOf<IXmlWriter>
     }
     virtual void outputUnicode(unsigned len, const UChar *field, const char *fieldname) override
     {
-        StringBuffer out;
-        outputXmlUnicode(len, field, nullptr, out);
+        char * buff = 0;
```

Comment:
This could be done a bit more efficiently (e.g. using functions based on junicode.hpp), but probably doesn't matter since unicode fields are not used that much.

Technically there should be a latin-1 -> utf 8 conversion in outputString, and if that was added (a separate PR) efficiency would matter more (but couldn't uses those jlib functions).